misc/python/materialize/cli/mzimage.py

```diff
@@ -84,6 +88,17 @@ def add_image_subcommand(name: str, **kwargs: Any) -> argparse.ArgumentParser:
         subparser.add_argument(
             "image", help="the name of an mzbuild image in this repository"
         )
+        subparser.add_argument(
+            "--build-mode",
+            default="release",
+            choices=["dev", "release"],
+            help="whether to build in dev or release mode",
+        )
+        subparser.add_argument(
+            "--coverage",
+            help="whether to enable code coverage compilation flags",
+            action="store_true",
```

Comment:
In `cli/mzcompose.py` we also add `default=None` here, but I imagine that's the, ahem, default anyways?

Reply:
Ah yeah, I wonder why I typed that `default=None` explicitly in mzcompose...